src/main/java/com/xiaomi/infra/pegasus/client/ClientOptions.java

```diff
@@ -369,6 +373,10 @@ public Builder credential(Credential credential) {
     public ClientOptions build() {
       return new ClientOptions(this);
     }
+
+    private static void validatePositiveNum(long value) {
+      assert value > 0 : String.format("must pass positive value: %d", value);
```

Comment:
assert value > 0 : String.format("must pass positive value: %ld", value);

Reply:
%d can be for int, long, byte in Java